README.md

```diff
@@ -85,6 +85,101 @@ Your minified file need to be appended with a comment directive to your source m
 //# sourceMappingURL=http://example.com/path/to/your/sourcemap.map
 ```
 
+## Setup for ReactJs
+
+### Initialization
+
+1. In the root `index.html` file which contains the application bundle, load the library before the `</head>` tag.
+
+```html
+<head>
+<!-- All of your other HTML <head> content -->
+<script src="https://cdn.rawgit.com/GoogleCloudPlatform/stackdriver-errors-js/v0.0.4/dist/stackdriver-errors-concat.min.js"></script>
+</head>
+```
+
+2. You can capture all exceptions with the following logic in your entry point, typically `index.js` with the following code:
+
+```javascript
+const errorHandler = new StackdriverErrorReporter(); // Note StackdriverErrorReporter is now a global variable
+errorHandler.start({
+  key: '<my-project-id>',
+  projectId: '<my-project-id>',
+  service: '<my-service>',              // (optional)
+  version: '<my-service-version>'       // (optional)
+});
+```
+
+3. If using webpack and the `DefinePlugin`, it is advisable to wrap the initialization logic to only occur in your production environment.  Otherwise, with local development you will receive 403s if you restricted your API key to your production href(which is _HIGHLY_ recommended).  The code for this would look something along these lines:
+
+```javascript
+// webpack.production.js
+// The rest of your webpack configuration
+// ...
+plugins: [
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify('production')
+    }),
+  // Your other plugins
+]
+// ...
+// The rest of your webpack configuration
+```
+
+```javascript
+// index.js
+const environment = process.env.NODE_ENV; 
+
+if (environment === 'production') {
+  const errorHandler = new StackdriverErrorReporter();
+  errorHandler.start({
+    key: '<my-project-id>',
+    projectId: '<my-project-id>',
+    service: '<my-service>',              // (optional)
+    version: '<my-service-version>'       // (optional)
+  });
+}    
+```
+
+4.  If you would like to use the logger throughout your application, there are many options that exist within the React ecosystem.  The simplest is to pull the initialization logic into its own file and reference it as necessary throughout your application.  An example would be as follows:
```

Comment:
Same here, this is not React specific. I suggest to to a dedicated section with point 3 and 4 about best practices for dev and prod environment.